From 7a831a347069166cc1e16300cdbf088cf55c88d6 Mon Sep 17 00:00:00 2001
Message-ID: <7a831a347069166cc1e16300cdbf088cf55c88d6.1705854493.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705854493.git.sharlatanus@gmail.com>
References: <cover.1705854493.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 14 Jan 2024 20:29:06 +0000
Subject: [PATCH 03/19] gnu: gpredict: Update home-page and description.

* gnu/packages/astronomy.scm (gpredict)[home-page]: Update it according
to project's README.
[description]: Add feature detailed list.

Change-Id: If5edc6e50796c7753ac47fac082c267145723361
---
 gnu/packages/astronomy.scm | 24 ++++++++++++++++++++++--
 1 file changed, 22 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index ab7663d7f5..47fc3c8fd9 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3196,6 +3196,8 @@ (define-public xplanet
     (license license:gpl2+)))
 
 (define-public gpredict
+  ;; The latest tag, 2.3, has no major difference with 2.2.1 and is dated for
+  ;; 2018. Additionally, there is some activity on the master branch.
   (package
     (name "gpredict")
     (version "2.2.1")
@@ -3222,14 +3224,32 @@ (define-public gpredict
      (list gettext-minimal intltool pkg-config))
     (inputs
      (list curl glib goocanvas gtk+))
-    (home-page "http://gpredict.oz9aec.net/index.php")
+    (home-page "https://oz9aec.dk/gpredict/")
     (synopsis "Satellite tracking and orbit prediction application")
     (description
      "Gpredict is a real-time satellite tracking and orbit prediction
 application.  It can track a large number of satellites and display their
 position and other data in lists, tables, maps, and polar plots (radar view).
 Gpredict can also predict the time of future passes for a satellite, and
-provide you with detailed information about each pass.")
+provide you with detailed information about each pass.
+
+Some core features of Gpredict include:
+
+@itemize
+@item Tracking of a large number of satellites only limited by the physical
+memory and processing power of the computer
+@item Display the tracking data in lists, maps, polar plots and any
+combination of these
+@item Have many modules open at the same either in a notebook or in their own
+windows.  The modules can also run in full-screen mode
+@item You can use many ground stations
+@item Predict upcoming passes
+@item Gpredict can run in real-time, simulated real-time (fast forward and
+backward), and manual time control
+@item Detailed information both the real time and non-real time modes
+@item Doppler tuning of radios via Hamlib rigctld
+@item Antenna rotator control via Hamlib rotctld
+@end itemize")
     (license license:gpl2+)))
 
 (define-public sgp4
-- 
2.41.0

